src/Controller/ProfilerController.php

```diff
@@ -52,6 +53,10 @@ public function __invoke(Request $request, string $token): Response
         $this->profiler->disable();
 
         $profile = $this->profiler->loadProfile($token);
+        // Type hint as int for the $limit argument of the find method was updated in Symfony 5.4.22 and 6.2.8
```

Comment:
Hi @aviator-ua.
Just add a `@phpstan-ignore-line` above line 57 and we can merge.
Thanks!